assets/app.js

```diff
@@ -984,6 +1002,52 @@ new Vue({
                 this.products[index].descriptions[language] = my_description
             }
         },
+        bulkChangeTranslationStatus: function (type) {
+            proceed = confirm("Are you sure that you want to set all " + this.type_dict[type]['name'] + " to be fully translated?")
+
+            if (proceed && hasApi) {
+                id_list = this[this.type_dict[type]['key']].map(function (a) {
+                    return a.id;
+                })
+
+                // ['1','2','3','4'] -> [['1','2'],['3','4']]: chunkSize = 2
+                var group_resources = [], chunkSize = 900;
+                for (var i = 0; i < id_list.length; i += chunkSize) {
+                    group_resources.push(id_list.slice(i, i + chunkSize));
+                }
+                _this = this
```

Comment:
I think there is no need for `_this` if we use
`resourceOptions = this.resourceDict[type]['selectableOptions']`
you will need to move up right before its first usage